sale_product_price/models/sale_order.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    pricelist_message = fields.Text()
+
+    @api.onchange('partner_id')
+    def _onchange_partner_id(self):
+        self._recompute_prices()
+
+    def action_update_prices(self):
+        super().action_update_prices()
+        self.message_post(body=self.pricelist_message)
+
+    @api.onchange('pricelist_id')
+    def _onchange_pricelist_id(self):
+        message = (f"you have change price list from"
+                   f" {self._origin.pricelist_id.name}"
+                   f" to {self.pricelist_id.name}.")
+        self.pricelist_message = message
```

Comment:
What is purpose of this field

Reply:
i have added message variable to avoid flake8 line length=79, now i have updated code, please review and suggest if changes to suggest.